pkg/transformer/openshift/openshift.go

```diff
@@ -129,13 +129,24 @@ func getComposeFileDir(inputFiles []string) (string, error) {
 // getAbsBuildContext returns build context relative to project root dir
 func getAbsBuildContext(context string, composeFileDir string) (string, error) {
 	cmd := exec.Command("git", "rev-parse", "--show-prefix")
-	cmd.Dir = composeFileDir
+	//absolute path of context
+	absContext, _ := filepath.Abs(context)
```

Comment:
`context` that is returned from libcompose is in some cases way wrong :-(
We are just creating messy workaround from something that is wrong in libcompose. I think this has to be fixed in libcompose - https://github.com/docker/libcompose/issues/449